### DISCUSSION TITLE
PRACTICAS

### DISCUSSION THREAD

**Discussion by iguelAL** [Launch-X-Latam/MisionFrontEnd#43] [Q&A]
Tengo unas dudas, ¿Como hago las practicas?, ¿en donde las hago?, ¿Como subo las practicas a un repositorio?, realmente soy nuevo en github y estoy un poco confundido, Gracias

**Comment by Emilio17MC** [comment]
Podrías comenzar a hacer las prácticas según los requerimientos que piden e irlas subiendo a tu repositorio de GitHub, es probable que a final de la misión Frontend se abra un formulario donde tengas que subir el link de tu repositorio como fue el caso del On Boarding y de esa forma puedas pasar a la misión Backend. Sobre la calificación, como se ha mencionado, aquí no se trabaja sobre un número, lo importante es que puedas ir construyendo un portafolio de desarrollo web, sin embargo, si quieres obtener feedback puedes publicar en el discussion de _Show & Tell_.

**Comment by iseddev** [comment]
Si estas iniciando desde las bases, te recomiendo:
- Estar atento a las clases en vivo
- Buscar algo de información básica del uso de Git (y de acuerdo al avance, de los otros temas relacionados)
- No querer avanzar más de lo debido, paso a paso llegarás al objetivo
Al principio puede parecer algo abrumador, pero con constancia y dedicación, le vas a ir agarrando el modo a cada tema.
Suerte y no te rindas!!!
